### PR TITLE
[ALPINAWEB-164] Add bottom up button

### PR DESCRIPTION
**Pull request by TheBreaken**

### CODE REVIEW

File: Resources/frontend/js/jquery.button-back-to-top.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+;(function ($, window) {
+
+    $.plugin('sdScrollTop', {
```
Comment (sd-lueckel) [inline Resources/frontend/js/jquery.button-back-to-top.js]:
```suggestion
    $.plugin('nlxScrollTop', {
```

File: Resources/frontend/js/jquery.button-back-to-top.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            });
+        },
+
+        destroy: function () {
+            var me = this;
+            $(me.$el).removeClass(me.opts.showCls);
+            me._destroy();
+        }
+    });
+    window.StateManager.addPlugin('body', 'sdScrollTop');
```
Comment (sd-lueckel) [inline Resources/frontend/js/jquery.button-back-to-top.js]:
```suggestion
    window.StateManager.addPlugin('body', 'nlxScrollTop');
```

**Review by sd-lueckel — CHANGES_REQUESTED** [review]

File: Resources/frontend/js/jquery.button-back-to-top.js (hunk 1 above)
Comment (TheBreaken) [reply Resources/frontend/js/jquery.button-back-to-top.js]: Done

File: Resources/frontend/js/jquery.button-back-to-top.js (hunk 2 above)
Comment (TheBreaken) [reply Resources/frontend/js/jquery.button-back-to-top.js]: Done

**Review by sd-lueckel — APPROVED** [review]

**Pull request merged**